packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -321,7 +321,7 @@ export default function MemberListItem(props: Props) {
               onSubmit={async (e) => {
                 e.preventDefault();
                 await signIn("impersonation-auth", {
-                  username: props.member.username || props.member.email,
+                  username: props.member.email,
```

Comment:
Always use emails since usernames are now none unique

Reply:
we are still using username here: [https://github.com/calcom/cal.com/blob/10-27-fix_impersonation-orgs/packages/features/users/components/UserTable/ImpersonationMemberModal.tsx#L30](https://github.com/calcom/cal.com/blob/10-27-fix_impersonation-orgs/packages/features/users/components/UserTable/ImpersonationMemberModal.tsx#L30
)
is it wanted?

Reply:
Good spot